pkgs/build-support/trivial-builders.nix

```diff
@@ -341,7 +341,9 @@ rec {
         if checkPhase == null then ''
           runHook preCheck
           ${stdenv.shellDryRun} "$target"
-          ${lib.getExe shellcheck} "$target"
+          # use shellcheck which does not include docs
+          # pandoc takes long to build and documentation isn't needed for in nixpkgs usage
+          ${lib.getExe (haskell.lib.compose.justStaticExecutables shellcheck.unwrapped)} "$target"
```

Comment:
I now partially understand the motivation for using `shellcheck.unwrapped` here, but I'm not convinced this is the best approach.
`(haskell.lib.compose.justStaticExecutables shellcheck.unwrapped)` is duplicated from the corresponding line in `pkgs/development/tools/shellcheck/default.nix`. Why not add something like `unwrapped-bin = haskell.lib.compose.justStaticExecutables shellcheck;` as a new passthru attribute over there?
That way `trivial-builders` also won't need the `haskell` argument.

Reply:
wrappers should usually have their unwrapped version in their passthru so i think this is fine the way it is